pilot/pkg/model/telemetry.go

```diff
@@ -1086,6 +1094,22 @@ var metricToPrometheusMetric = map[string]string{
 	"GRPC_RESPONSE_MESSAGES": "response_messages_total",
 }
 
+func metricRotationInterval() *durationpb.Duration {
+	if features.MetricRotationInterval == 0*time.Second {
+		return nil
+	}
+
+	return durationpb.New(features.MetricRotationInterval)
+}
+
+func metricGracefulDeletionInterval() *durationpb.Duration {
+	if features.MetricGracefulDeletionInterval == 5*time.Minute {
```

Comment:
this doesn't allow a user to explicitly set to 5min. Can handle this with .Lookup instead of .Get

Reply:
default value is 5m, I think this will reduce this size of xds.
https://github.com/istio/proxy/blob/b4a374ad97bb50e8f80c64f70bee5bdd85cbcaf5/source/extensions/filters/http/istio_stats/config.proto#L120